### PR TITLE
Don't install htcondor-ce-view in the base image (to keep it out of the hosted-ce image)

### PR DESCRIPTION
**Pull request by matyasselmeci**
SOFTWARE-5147

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -51,7 +50,8 @@ ARG BASE_YUM_REPO=release
 LABEL maintainer "OSG Software <help@opensciencegrid.org>"
 LABEL name "osg-ce-condor"
 
-RUN yum install -y osg-ce-condor && \
+RUN yum install -y osg-ce-condor \
+                   htcondor-ce-view && \
```
Comment (brianhlin) [inline Dockerfile:54]: We don't need it here either, let's rip it out completely

**Review by brianhlin — APPROVED** [review]
One change requested, otherwise LGTM

File: Dockerfile (hunk 1 above)
Comment (matyasselmeci) [reply Dockerfile:54]: Do we know that? We run the hosted CEs so we know what we want there, but the osg-ce-condor image is being used by non-OSG staff...

Comment (brianhlin) [reply Dockerfile:54]: Who? AFAIK, the Open Science CE is the only service that uses this

Comment (matyasselmeci) [reply Dockerfile]: OK

**Pull request merged**